src/pages/trade/[[...pair]].tsx

```diff
@@ -0,0 +1,72 @@
+import { useRouter } from "next/router";
```

Comment:
haven't come across double bracket file names or see anything in docs, is this something in a new version of next?